src/studio/src/designer/frontend/packages/schema-editor/src/components/Editor.test.tsx

```diff
@@ -54,35 +55,35 @@ const renderEditor = async (customState?: Partial<ISchemaState>) => {
     </Provider>,
   );
 
-  return [mockStore];
+  return { store, user };
 };
 
-const clickOpenAddMenuButton = () =>
-  fireEvent.click(
+const clickOpenAddMenuButton = (user: UserEvent) =>
+  user.click(
     screen.getByRole('button', {
       name: mockLanguage.schema_editor.add,
     }),
   );
-const clickAddMenuItem = (name: string) =>
-  fireEvent.click(screen.getByRole('menuitem', { name }));
+const clickAddMenuItem = (user: UserEvent, name: string) =>
+  user.click(screen.getByRole('menuitem', { name }));
 
-const clickOpenContextMenuButton = () =>
-  fireEvent.click(screen.getByTestId('open-context-menu-button'));
+const clickOpenContextMenuButton = (user: UserEvent) =>
+  user.click(screen.getByTestId('open-context-menu-button'));
```

Comment:
Shouldnt these be async, and the `user.click` should be awaited?

Reply:
Disse funksjonene returnerer ett promise som seinere blir awaitet, så dette er korrekt kode.